Reference/Searching/Examine/quick-start.md

```diff
@@ -55,74 +56,54 @@ In MVC we have a method called `TypedSearch` on the `UmbracoHelper` which will r
                 </li>
             }
         </ul>
-    }   
+    }
 
-### Razor
+### ExamineManager
 
-In razor macros there's a `Search` method on the `DynamicNode` model which will return a `DynamicNodeList`:
+To complete more complex searches, we can use the ExamineManager and our custom search provider to expose further functionality, such as being able to handle mispellings, for example.
 
-    @if (!string.IsNullOrEmpty(Request.QueryString["query"]))    
+    var query = Request.QueryString["query"];
+
+    var searcher = Examine.ExamineManager.Instance.SearchProviderCollection["ExternalSearcher"];
+    
+    // the boolean parameter is whether to use wildcards when searching.
+    var searchResults = Searcher.Search(query, true).OrderByDescending(x => x.Score);
```

Comment:
I think `Searcher.Search` should be `searcher.Search`

Reply:
I think using OrderByDescending is a bad practice because that would enumerate all results. Except if there is a OrderByDescending extentionmethod from Examine.  But then we need to mention the namespace in the example.

Reply:
Good point - there is an extension method within Examine, so I've included the `using` in the two relevant examples.